makefile

```diff
@@ -35,8 +35,18 @@ omim.owl: omim.ttl mondo_exactmatch_omim.sssom.owl mondo_exactmatch_omimps.sssom
 		query --update sparql/hgnc_links.ru \
 		convert -f ofn -o $@
 
-mondo_genes.csv: omim.owl
+mondo_genes.robot.tsv: omim.owl
+	# Create a TSV of relational information for gene and disease classes
 	robot query -i omim.owl --query sparql/mondo_genes.sparql $@
+	# Insert the source_code column as the second to last column
+	awk 'BEGIN {FS=OFS="\t"} {if (NR==1) {$$(NF+1)=$$(NF); $$(NF-1)="?source_code";} else {$$(NF+1)=$$(NF); $$(NF-1)="MONDO:OMIM";}} 1' $@ > temp_file && mv temp_file $@
+	# Remove the first character of each field in the header
+	awk 'BEGIN {FS=OFS="\t"} NR==1 {for (i=1; i<=NF; i++) $$i=substr($$i, 2)} {print}' $@ > temp_file && mv temp_file $@
+	# Remove < and > characters from specified columns
+	awk 'BEGIN {FS=OFS="\t"} NR>1 {gsub(/^<|>$$/, "", $$1); gsub(/^<|>$$/, "", $$2); gsub(/^<|>$$/, "", $$5)} {print}' $@ > temp_file && mv temp_file $@
+	# Insert ROBOT subheader
+	robot_subheader="ID\tSC 'has material basis in germline mutation in' some %\t>A oboInOwl:source\t>A oboInOwl:source\t" && \
+	sed 1a"$$robot_subheader" $@ > temp_file && mv temp_file $@
```

Comment:
## `hgnc_id `: `SC 'has material basis in germline mutation in' some %`
What Nico wrote in the issue was a placeholder for the actual thing. I went and looked through some examples we had of this pattern `SC '<PROPERTY>' some %`, and also found the correct string representation `'has material basis in germline mutation in'`. I'm basing it off of several different locations in `mondo` where I saw this: `'%s and ''has material basis in germline mutation in'' some %s'`